os_update-formula/os-update/init.sls

```diff
@@ -59,16 +59,24 @@ os-update_config_file:
     - name: /etc/os-update.conf
 {%- endif %}
 
-{%- if config.time %}
+{%- if config.time or 'accuracysec' in config or 'randomizeddelaysec' in config %}
```

Comment:
You'll need a similar change for the watch: in line +88

Reply:
Thanks for spotting, to avoid additional logic there I replaced it with a simpler watch_in.